src/main/java/gov/cms/madie/services/HumanReadableService.java

```diff
@@ -126,7 +126,7 @@ String getCmsId(Measure measure) {
   HumanReadableMeasureInformationModel buildMeasureInfo(Measure measure) {
     boolean patientBased = false;
     String measureScoring = "";
-    if (!measure.getGroups().isEmpty()) {
+    if (measure.getGroups() == null || !measure.getGroups().isEmpty()) {
```

Comment:
can we do CollectionUtils.isEmpty(measure.getGroups()) I think it will check both null and empty condition